instrumentation/google.golang.org/grpc/otelgrpc/README.md

```diff
@@ -0,0 +1,20 @@
+# gRPC instrumentation
+
+This library is the instrumentation library for `google.golang.org/grpc`
+
+For now you can instrument your program which use `google.golang.org/grpc` in two ways:
+
+- by gRPC Interceptors
+- [experimental] by gPRC `stats.Handler`
```

Comment:
1. typo
2. what is it noted as "experimental"?

```suggestion
- by gRPC `stats.Handler`
```

Reply:
> 2\. what is it noted as "experimental"?

I wrote "experimental" to express that the feature is now not stable enough, and should be used for a period of time to reach the stable stage.

Do we have some common conventions to express this?
@pellared

Reply:
> It would be better if we put the docs in `doc.go` instead of `README.md`. More:

I will move to doc.go

Reply:
> I wrote "experimental" to express that the feature is now not stable enough, and should be used for a period of time to reach the stable stage.

The version of the module is `v0.*` 😉

Reply:
> conventions

Got it

Reply:
I already moved readme to doc.go

Reply:
Lint is failing (probably you need to format `doc.go`)

Reply:
> Lint is failing (probably you need to format `doc.go`)

Done